inputfiles/overridingTypes.json

```diff
@@ -2318,6 +2337,58 @@
                         }
                     ]
                 }
+            },
+            "BroadcastChannel": {
+                "name": "BroadcastChannel",
+                "methods": {
+                    "method": {
+                        "postMessage": {
+                            "name": "postMessage",
+                            "override-signatures": [
+                                "postMessage(message: any): void"
+                            ]
+                        }
+                    }
+                }
+            },
+            "MessagePort": {
+                "name": "MessagePort",
+                "methods": {
+                    "method": {
+                        "postMessage": {
+                            "name": "postMessage",
+                            "override-signatures": [
+                                "postMessage(message: any, transferList?: Transferable[]): void"
```

Comment:
Why `transferList` instead of `transfer` here?

Reply:
I named it `transferList` because it is landed so in the MDN docs.

https://developer.mozilla.org/en-US/docs/Web/API/Worker/postMessage

I can change it to `transfer` if it's preferred.

Reply:
I changed it to `transfer`.